File: src/main/scala/com/example/scala/Pessoas.scala
package com.example.scala

/**
 * @author Giovanni Silva
 *         09/03/15.
 */
object Pessoas {
  def main(args: Array[String]): Unit = {
    val lista = PessoaOperations.gerarLista(7)
    PessoaOperations.imprimirLista(lista)
  }
}

File: src/main/java/com/example/java/Pessoas.java
package com.example.java;

import java.util.List;

/**
 * @author Giovanni Silva
 *         09/03/15.
 */
public class Pessoas {
    public static void main(String[] args) {
        List<Pessoa> pessoaList = PessoaOperations.gerarLista(5);
        PessoaOperations.imprimirLista(pessoaList);
    }
}
